src/style.css

```
@@ -1 +1,48 @@
 /* add styles here */
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('images/star.svg');
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:not(:nth-child(n + 1)){
+    background-image: url('images/star-active.svg');
+}
+
+.stars--1 .stars__star:not(:nth-child(n + 2)){
+    background-image: url('images/star-active.svg');
+}
+
+.stars--2 .stars__star:not(:nth-child(n + 3)){
+    background-image: url('images/star-active.svg');
+}
+
+.stars--3 .stars__star:not(:nth-child(n + 4)){
+    background-image: url('images/star-active.svg');
+}
+
+.stars--4 .stars__star:not(:nth-child(n + 5)){
+    background-image: url('images/star-active.svg');
+}
+
+.stars--5 .stars__star:not(:nth-child(n + 6)){
+    background-image: url('images/star-active.svg');
+}
```

```suggestion
.stars--0 .stars__star:not(:nth-child(n + 1)),
.stars--1 .stars__star:not(:nth-child(n + 2)),
.stars--2 .stars__star:not(:nth-child(n + 3)),
.stars--3 .stars__star:not(:nth-child(n + 4)),
.stars--4 .stars__star:not(:nth-child(n + 5)),
.stars--5 .stars__star:not(:nth-child(n + 6)){
    background-image: url('images/star-active.svg');
}
```
combine it using a comma